coreservices/iam/themes/catenax-central/email/html/template.ftl

```diff
@@ -3,25 +3,27 @@
   <head>
   </head>
     <body text="#000000" bgcolor="#FFFFFF">
-    <#if realmName??>
+    
     <table width="100%" bgcolor="#FFFFFF" cellspacing="0" cellpadding="15"><tr><td>
     <font style="font-size:24px;" size="5" color="#000000"><b>
       <#nested "Welcome">
       </b></font>
-      <p></p>
+      <</br>
```

Comment:
the correct syntax for a br tag without children is `<br />` . this one here also seems to start with `<<`.
this applies to all br-Tags in the file.

Reply:
done, exchanged with <br />